x-pack/plugins/apm/server/plugin.ts

```diff
@@ -83,20 +80,34 @@ export class APMPlugin implements Plugin<APMPluginContract> {
           metricsIndices: this.currentConfig['apm_oss.metricsIndices'],
           onboardingIndices: this.currentConfig['apm_oss.onboardingIndices'],
           sourcemapIndices: this.currentConfig['apm_oss.sourcemapIndices'],
-          transactionIndices: this.currentConfig['apm_oss.transactionIndices'],
-        },
+          transactionIndices: this.currentConfig['apm_oss.transactionIndices']
+        }
       })
     );
 
+    const usageCollection = plugins.usageCollection;
```

Comment:
the usage collector is now registered from the plugin setup, but it can't block setup, since it depence on core start services, so it uses regular old Promise API.